src/libs/httpclient/creporteruploaditem.cpp

```diff
@@ -48,13 +48,13 @@ const char *status_string[] = {"Waiting", "Sending", "Error", "Finished", "Cance
   */
 class CReporterUploadItemPrivate
 {
-    public:
-        QString filepath;
-        QString filename;
-        QString errorString;
-        qint64 filesize;
-        CReporterHttpClient *http;
-        CReporterUploadItem::ItemStatus status;
+public:
+    QString filepath;
+    QString filename;
+    QString errorString;
+    qint64 filesize;
+    CReporterHttpClient *http;
```

Comment:
Not related to this astyle commit per se but... While this get initialized to a null pointer in CReporterUploadItem constructor, a default constructor for private class would imo be cleaner.